spec/models/concerns/countable_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe Countable do
+  before(:all) do
+    FactoryGirl.create(:post)
+    @post = Post.last
+  end
+
+  it "should has_many action_counts" do
+    expect(Post.new).to have_many(:click_counts).class_name('Count')
+      .dependent(:destroy)
+      .inverse_of(:countable)
+  end
+
+  it "should define incr_action_count correctly" do
+    expect { @post.incr_click_count }.to change { @post.click_counts.count }
+      .by(1)
+    expect(@post.click_counts.first.countable).to eq(@post)
```

Comment:
```ruby
expect { @post.incr_click_count }.to \
   change { @post.click_counts.count }.by(1)
```

Since `change ... by 1` serves an independent concept, better put them in the same line for readability.